deployment/src/main/java/io/quarkiverse/openapi/generator/deployment/codegen/OpenApiGeneratorCodeGenBase.java

```diff
@@ -1,18 +1,6 @@
 package io.quarkiverse.openapi.generator.deployment.codegen;
 
-import static io.quarkiverse.openapi.generator.deployment.CodegenConfig.EXCLUDE_FILES;
-import static io.quarkiverse.openapi.generator.deployment.CodegenConfig.INCLUDE_FILES;
-import static io.quarkiverse.openapi.generator.deployment.CodegenConfig.INPUT_BASE_DIR;
-import static io.quarkiverse.openapi.generator.deployment.CodegenConfig.VALIDATE_SPEC_PROPERTY_NAME;
-import static io.quarkiverse.openapi.generator.deployment.CodegenConfig.VERBOSE_PROPERTY_NAME;
-import static io.quarkiverse.openapi.generator.deployment.CodegenConfig.getAdditionalModelTypeAnnotationsPropertyName;
-import static io.quarkiverse.openapi.generator.deployment.CodegenConfig.getBasePackagePropertyName;
-import static io.quarkiverse.openapi.generator.deployment.CodegenConfig.getCustomRegisterProvidersFormat;
-import static io.quarkiverse.openapi.generator.deployment.CodegenConfig.getImportMappingsPropertyName;
-import static io.quarkiverse.openapi.generator.deployment.CodegenConfig.getReturnResponsePropertyName;
-import static io.quarkiverse.openapi.generator.deployment.CodegenConfig.getSanitizedFileName;
-import static io.quarkiverse.openapi.generator.deployment.CodegenConfig.getSkipFormModelPropertyName;
-import static io.quarkiverse.openapi.generator.deployment.CodegenConfig.getTypeMappingsPropertyName;
+import static io.quarkiverse.openapi.generator.deployment.CodegenConfig.*;
```

Comment:
Yeah was wondering about that. I use IDEA and it just folds those above a certain number. Guess the standard formatter plugin doesn't solve these. Do you want it unfolded?

Reply:
@bpasson Yes, please.
To avoid that you can configure IDEA to fold only when the class count is bigger than 999.
![image](https://user-images.githubusercontent.com/11776454/236276010-b44f3f86-b90c-4daa-b7c8-f831667148ed.png)

@ricardozanini we could use the same as Quarkus: https://github.com/quarkusio/quarkus/blob/main/CONTRIBUTING.md#ide-config-and-code-style

Reply:
Can you add a note to the contributions guide?